### PR TITLE
Dynamic page button

### PR DESCRIPTION
**Pull request by Faberen**

### CODE REVIEW

File: src/Chirp.Infrastructure/CheepRepository.cs [hunk 1]
```diff
@@ -110,5 +110,45 @@ public List<CheepDTO> GetCheepsFromAuthor(string author, int? pageNum)
...
+        return cheepsToReturn.Count();
+    }
+
+    public int GetCountOfAllCheepFromAuthor(string author)
+    {
+        //Creates a list of max 32 CheepDTO sorted by recent cheep and only for the given author
+
+        List<CheepDTO> cheepsToReturn = new List<CheepDTO>();
+
+        var cheepsDTO = db.Cheeps.OrderByDescending(c => c.TimeStamp.Ticks)
```
Comment (Lukan707) [inline src/Chirp.Infrastructure/CheepRepository.cs:139]: Could we perphaps request the count, via an aggregate, function directly instead.
If we imageine we have thousands of cheeps, a lot of data is sent between the layers, were by returning the count only, a constant low amount of data is always sent.

**Review by Lukan707 — CHANGES_REQUESTED** [review]
Nice work.
It is a cool feature, that we can flex with 💯

File: src/Chirp.Infrastructure/CheepRepository.cs
Comment (annahj1006) [inline src/Chirp.Infrastructure/CheepRepository.cs:1]: Maybe add a comment about why we need the count of cheeps. The functionality of the methods is a bit confusing, i get that they are working with the pages. But maybe make it a bit more clear.

**Review by annahj1006 — APPROVED** [review]

File: src/Chirp.Infrastructure/CheepRepository.cs
Comment (Faberen) [reply src/Chirp.Infrastructure/CheepRepository.cs:1]: Added comments.

File: src/Chirp.Infrastructure/CheepRepository.cs (hunk 1 above)
Comment (Faberen) [reply src/Chirp.Infrastructure/CheepRepository.cs]: The count now comes from the SQL query as requested

**Review by annahj1006 — APPROVED** [review]

**Review by Lukan707 — APPROVED** [review]
Great. Really nice

**Pull request merged**